### PR TITLE
Add setting for message list density

### PR DESCRIPTION
**Pull request by cketti**
### Message list density setting
image

### Compact
image

### Default
image

### Relaxed

image

### CODE REVIEW

File: app/core/src/main/java/com/fsck/k9/K9.kt [hunk 1]
```diff
@@ -152,6 +152,9 @@ object K9 : EarlyInit {
     @JvmStatic
     var lockScreenNotificationVisibility = LockScreenNotificationVisibility.MESSAGE_COUNT
 
+    @JvmStatic
```
Comment (wmontwe) [inline app/core/src/main/java/com/fsck/k9/K9.kt:155]: This class is marked as deprecated and `GeneralSettings` should be used instead. Is this still true?

**Review by wmontwe — COMMENTED** [review]
At the moment all configuration is derived from one central location. Wouldn't it make sense to move this kind of configuration close to the feature?

File: app/core/src/main/java/com/fsck/k9/K9.kt (hunk 1 above)
Comment (cketti) [reply app/core/src/main/java/com/fsck/k9/K9.kt:155]: Probably not. Moving to `GeneralSettings` would just get rid of the global `K9` object, but keep the central location for all settings.

> At the moment all configuration is derived from one central location. Wouldn't it make sense to move this kind of configuration close to the feature?

Yes. However, we still want to use a central settings storage. We need to come up with a proper architecture for how to do that first.

**Review by wmontwe — APPROVED** [review]
LGTM

**Pull request merged**